README.md

```diff
@@ -7,61 +7,99 @@
 </a> <a href="https://goreportcard.com/report/sigs.k8s.io/kind"><img alt="Go Report Card" src="https://goreportcard.com/badge/sigs.k8s.io/kind" /></a>
 
 
-`kind` is a tool for running local Kubernetes clusters using Docker container "nodes".  
-`kind` is primarily designed for testing Kubernetes 1.11+, initially targeting the [conformance tests].
+`kind` is a tool for running local Kubernetes clusters using Docker container as "nodes".  
+`kind` is primarily designed for testing Kubernetes 1.12+.
+It initially targets the [conformance tests].
 
-If you have [go] and [docker] installed `go get sigs.k8s.io/kind && kind create cluster` is all you need!
+If you have [Go] and [Docker] installed, the following commands are all you need!
 
-<img src="https://gist.githubusercontent.com/BenTheElder/621bc321fc6d9506fd936feb36d32dd0/raw/7fe14e9d0929cab428929ca6c501abc990c07359/kind-create-cluster.gif" alt="2x speed `kind create cluster` demo" />
+```console
+$ go get -u sigs.k8s.io/kind
```

Comment:
i think we can omit `$` in all bash / console examples.
this is at least what the k/website guide recommends.